src/coap_blockwise.h

```diff
@@ -3,6 +3,12 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+
+#ifdef __cplusplus
```

Comment:
Is this also required in the private headers? Or only in the public headers? These headers in `src/` aren't available to user applications, so they will only be included by C files, not C++.

Reply:
Will remove from the private headers in `src/`